src/components/PlantingSites/PlantingSitesCellRenderer.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { makeStyles } from '@mui/styles';
+import CellRenderer, { TableRowType } from '../common/table/TableCellRenderer';
+import { RendererProps } from '../common/table/types';
+
+const useStyles = makeStyles(() => ({
+  text: {
+    fontSize: '14px',
+    '& > p': {
+      fontSize: '14px',
+    },
+  },
+}));
+
+export default function PlantingSitesCellRenderer(props: RendererProps<TableRowType>): JSX.Element {
+  const classes = useStyles();
+  const { column, row, value, index } = props;
+
+  if (column.key === 'name') {
+    return <CellRenderer index={index} column={column} value={value} row={row} className={classes.text} />;
+  }
+
+  return <CellRenderer {...props} className={classes.text} />;
```

Comment:
line 23 does the same thing as line 20, we can maybe remove lines 19 - 21

Reply:
Yes, I just leave this prepared for adding the link to the name column, but I will remove it and add it back later